lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -913,7 +913,7 @@ fc_extras
         # Set the cube global attributes. 
         for attr_name, attr_value in six.iteritems(cf_var.cf_group.global_attributes):
             try:
-                if isinstance(attr_value, unicode):
+                if isinstance(attr_value, six.text_type):
```

Comment:
Hrm, good point. I sort of ignored what was in the `if`, but it makes sense to be explicit for when Python2 finally dies in a million years.